### DISCUSSION TITLE
How do I migrate from 1Password 8?

### DISCUSSION THREAD

**Discussion by DaveLeonard** [keepassxreboot/keepassxc#8449] [Q&A] [answered]
Hi Everybody,

I was looking for a way to migrate from 1Password 8 export file .1pux to KeePass.
Apprently .cvs doesn't keep OTP informations and I need to keep them.

Do you know how to do that keeping the OTP informations?

Thank you! David

**Comment by phoerious** [comment, marked as answer]
You have to export it as .opvault, which you can then import into KeepassXC. We do not support .1pux.
See https://keepassxc.org/docs/KeePassXC_UserGuide.html#_importing_1password_opvault

**Reply by DaveLeonard** [reply]
Thanks, but this feature has been removed from 1Password 8, there is no way that I'm aware of to export database in .opvault any more.

**Reply by phoerious** [reply]
I think you have to open it in 1Password 7 then.

**Reply by DaveLeonard** [reply]
Unfortunately it seems that 1Password 8 database is not compatible anymore with 1Password 7 app.
I'll try it anyway.

**Reply by DaveLeonard** [reply]
If it's the case, it will be much harder to switch from 1Password to KeePass, which is a bad thing IMO.

**Reply by phoerious** [reply]
There is a feature request for supporting the new format. https://github.com/keepassxreboot/keepassxc/issues/7545

**Reply by DaveLeonard** [reply]
That so great thanks @phoerious
